bigbite/themer
# [#113] - Radius not an option

## Description

[#113](https://github.com/bigbite/themer/issues/113) - This PR aims to introduce a `border radius` option alongside the current border option.

Using the experimental `BorderRadiusControl` [component](https://github.com/WordPress/gutenberg/tree/8dd99ede394c871a94bafc33a71abb0cf24c7bc8/packages/block-editor/src/components/border-radius-control), a border radius can be applied using a singular value or 'unlinked' values for each corner of the element.

This PR also addresses a slight mis-match in label `font-weight` by changing the value assigned to `.themer--styles__item__label` from `600` to `500`.

## Change Log

- `src/editor/components/StylesBorder.js` - Including radius component in the layout
- `src/editor/components/StylesBorderRadius.js` - The new component to handle the radius values with the `BorderRadiusControl` component
- `src/editor/styles/components/styles.scss` - Updates the custom label font weight

## Steps to test

- Navigate to the Styles Editor page
- Select an element or block, such as `Button`
- Scroll down in the styles panel to the bottom where `Border and radius` is in view
- Adjust the `Radius` Linked value using the input box
- When the `Visual` view is selected, notice the border radius will change accordingly
- When the `Code` view is selected, notice the value of the radius property will change accordingly
- Adjust the `Radius` Linked value using the range slider
- Notice the changes are reflected in both views
- Click on the `Unlink` button to reveal 4 input boxes, one for each corner of the element
- Update the value in each box
- Notice the changes are reflected in both the `Visual` and `Code` views
- Follow these steps on other blocks and elements

## Screenshots/Videos

~~[A demo showing various interactions with the radius setting and the visual preview](http://bigbite.im/v/BJv9HP)~~
~~[A demo showing various interactions with the radius setting and the code preview](http://bigbite.im/v/Otc2gG)~~
[An updated demo showing the border radius setting in visual preview](http://bigbite.im/v/TUdljv)
[An updated demo showing the border radius setting in code preview](http://bigbite.im/v/so63D4)

## Checklist:
